pkg/pgmodel/upgrade_tests/upgrade_test.go

```diff
@@ -421,7 +444,7 @@ func toPreviousVersion(version *semver.Version) {
 	version.Pre = nil
 
 	if version.Patch > 0 {
-		version.Patch -= 1
+		version.Patch = 0
```

Comment:
This is not correct in general. But I think we are gonna change this logic anyway and this is needed for the PR so I think its ok